### ISSUE TITLE
[Enhancement]: Change the default value for payload_format_version in apigatewayv2 resource

### ISSUE THREAD

**Issue by leandrodamascena** [hashicorp/terraform-provider-aws#37035]
### Description

According to the AWS documentation, [apigatewayv2](https://docs.aws.amazon.com/apigateway/latest/developerguide/http-api-develop-integrations-lambda.html#http-api-develop-integrations-lambda.proxy-format) uses the latest version of the payload by default, which is 2.0, but Terraform still maintains 1.0. 

> If you don't specify a payload format version, the AWS Management Console uses the latest version by default. 

It can confuses customers that want the same experience using AWS Console and Terraform.

Thanks

### Affected Resource(s) and/or Data Source(s)

aws_apigatewayv2_integration

### Potential Terraform Configuration

```terraform
https://github.com/hashicorp/terraform-provider-aws/blob/f2528d3edb33094a39038d34a4b06d0535c14bdc/internal/service/apigatewayv2/integration.go#L117
```

### Would you like to implement a fix?

Yes

**Comment by leandrodamascena** [comment]
Hi @justinretzolk! I would like to submit a PR changing the default version. This is my first time contributing to the project and I just wanted to make sure you're okay with it.

I will change the default version and documentation.

Thanks

**Comment by AdamTylerLynch** [comment]
Yes, the Terraform provider is open source and we welcome contributions! 

We at AWS have published a workshop on making contributions:

https://catalog.workshops.aws/contributing-to-the-terraform-provider/en-US

For this one in particular, if you think only a documentation update is needed, you can follow this part of ther workshop:

https://catalog.workshops.aws/contributing-to-the-terraform-provider/en-US/30-development-process/70-creating-documentation
